src/main/kotlin/com/kotlin/boardproject/domain/comment/service/CommentService.kt

```diff
@@ -1,16 +1,16 @@
 package com.kotlin.boardproject.domain.comment.service
 
-import com.kotlin.boardproject.domain.comment.dto.BlackCommentRequestDto
-import com.kotlin.boardproject.domain.comment.dto.BlackCommentResponseDto
-import com.kotlin.boardproject.domain.comment.dto.CancelLikeCommentResponseDto
-import com.kotlin.boardproject.domain.comment.dto.CreateCommentRequestDto
-import com.kotlin.boardproject.domain.comment.dto.CreateCommentResponseDto
-import com.kotlin.boardproject.domain.comment.dto.DeleteCommentRequestDto
-import com.kotlin.boardproject.domain.comment.dto.DeleteCommentResponseDto
-import com.kotlin.boardproject.domain.comment.dto.LikeCommentResponseDto
-import com.kotlin.boardproject.domain.comment.dto.MyCommentResponseDto
-import com.kotlin.boardproject.domain.comment.dto.UpdateCommentRequestDto
-import com.kotlin.boardproject.domain.comment.dto.UpdateCommentResponseDto
+import com.kotlin.boardproject.domain.comment.dto.black.BlackCommentRequestDto
```

Comment:
🚫 **[ktlint]** <import-ordering> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Imports must be ordered in lexicographic order without any empty lines in-between with "java", "javax", "kotlin" and aliases in the end